api/server/server.go

```diff
@@ -21,6 +21,20 @@ type Route struct {
 	fn   func(http.ResponseWriter, *http.Request)
 }
 
+type AnonRegex struct {
+	anonymizeRegx *regexp.Regexp
+	replaceString string
+}
+
+// anonTokenRegxes[] used to hide strings when logging.
+var anonRegxes = []AnonRegex{
+	{
+		// Anonymize the token=.*, replace with token=***..*,
+		anonymizeRegx: regexp.MustCompile(`token=.*,`),
```

Comment:
should this be `token=.*,*` in case there's no comma after? i.e. a log statement `could not find token=abcd`. 

Maybe not, just unfamiliar with what all logs look like.

Reply:
This actually is specific to the token in the 'ID=' part of the output log string.    So the format I followed would be for that one.   Which is described in the bug.   However I can see that if you wanted to capture everything included after the token you could add that regex but then having both regex's one could overdue the other.  I guess this should not be this generic and should be relegated to the types of fields..   Like this one should be use for IDs and if we wanted we could create another []AnonRegex for Msgs.. I guess.   I will rename this for now to anonIDRegxes...